lib/id3c/labelmaker.py

```diff
@@ -240,6 +246,7 @@ class SamplesHaarviLayout(LabelLayout):
     "collections-household-observation-asymptomatic": CollectionsHouseholdObservationAsymptomaticLayout,
     "collections-household-intervention": CollectionsHouseholdInterventionLayout,
     "collections-household-intervention-asymptomatic": CollectionsHouseholdInterventionAsymptomaticLayout,
+    "collections-household-general-season-three": CollectionsHouseholdGeneralSeasonThreeLayout,
```

Comment:
The name of the identifier set and the class are a mouthful 😆 
Maybe we can get remove season three from the name?

Reply:
I added season three in there explicitly because we now have 5 household related identifier sets. I tried to make this easier on myself when I have to find the right identifier set next time. OK?

Reply:
I agree with Jover here. I don't think it's necessary to qualify a barcode set with a year. IIUC, the only reason we're adding a new identifier set is because they're dropping the observation vs. intervention distinction. Therefore, `collections-household-general` should be a sufficient distinction.

Reply:
Or even, `collections-household`.

Reply:
OK. I lost that battle.